cnv_somatic_panel_workflow.b37.inputs.json

```diff
@@ -0,0 +1,48 @@
+{
+  "##_COMMENT1": "Inputs",
+  "CNVSomaticPanelWorkflow.normal_bams": ["gs://gatk-test-data/cnv/somatic/SM-74NEG.bam", "gs://gatk-test-data/cnv/somatic/SM-74NF5.bam"],
+  "CNVSomaticPanelWorkflow.normal_bais": ["gs://gatk-test-data/cnv/somatic/SM-74NEG.bam.bai", "gs://gatk-test-data/cnv/somatic/SM-74NF5.bam.bai"],
+  "CNVSomaticPanelWorkflow.pon_entity_id": "wes-do-gc",
+
+  "##_COMMENT2": "Reference and Resource",
+  "CNVSomaticPanelWorkflow.ref_fasta_dict": "gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict",
+  "CNVSomaticPanelWorkflow.ref_fasta": "gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta",
+  "CNVSomaticPanelWorkflow.ref_fasta_fai": "gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai",
+  "CNVSomaticPanelWorkflow.intervals": "gs://gatk-test-data/cnv/somatic/ice_targets.tsv.interval_list",
+  
+  "##_COMMENT3": "Docker",
+  "CNVSomaticPanelWorkflow.gatk_docker": "broadinstitute/gatk:4.0.0.0",
+
+  "##_COMMENT4": "Disk Size Optional",
+  "##CNVSomaticPanelWorkflow.AnnotateIntervals.disk_space_gb": "",
```

Comment:
Why are these optional parameters formatted differently than those in the pair workflow?